openassessment/xblock/static/js/src/oa_server.js

```diff
@@ -705,6 +705,30 @@ export class Server {
       });
     });
   }
+
+  /**
+   * Clone a rubric into ORA from an existing rubric
+   * @param {xblock-id} rubricLocation, xblock locator for ORA to clone rubric from
+   * @returns {promise} a JQuery Promise which will resolve with rubric data
+   */
+  cloneRubric(rubricLocation) {
+    const url = this.url('get_rubric');
+    const payload = { target_rubric_block_id: String(rubricLocation) };
+
+    return $.Deferred((defer) => {
```

Comment:
Investigate replacing with `axios` or `fetch`

Reply:
Ummm... I'm gonna call this out of scope since everything else here uses this format.

Reply:
new ticket for replacing network comms may make sense, or at least cataloguing them